grammar/haxe.bnf

```diff
@@ -301,7 +301,7 @@ enumConstructorParameters ::= '(' parameterList ')' {pin=1}
 
 varDeclaration ::= (macroMember | declarationAttribute)* 'var' varDeclarationPartList ';'
 {pin=3 mixin="com.intellij.plugins.haxe.lang.psi.impl.HaxePsiFieldImpl" implements="com.intellij.plugins.haxe.lang.psi.HaxePsiField"}
-private varDeclarationPartList ::= varDeclarationPart (',' varDeclarationPart)*
+private varDeclarationPartList ::= varDeclarationPart
```

Comment:
Doesn't this change mean that you can no longer declare:  `var a,b : Int;`  ...Because the `,b` is no longer allowed?